components/setup/CodeWithCopyButton.tsx

```diff
@@ -5,20 +5,29 @@ import { MdContentCopy } from "react-icons/md";
 
 interface CodeWithCopyButtonProps {
 	userId: string;
+	selectedProvider: string;
+	selectedInstallationType: string;
 }
 
-const CodeWithCopyButton: React.FC<CodeWithCopyButtonProps> = ({ userId }) => {
+const CodeWithCopyButton: React.FC<CodeWithCopyButtonProps> = ({ userId, selectedInstallationType, selectedProvider }) => {
 	const [isCopied, setIsCopied] = useState<boolean>(false);
 	const [isButtonDisabled, setIsButtonDisabled] = useState<boolean>(false);
 	const [selfHostingCode, setSelfHostingCode] = useState<string>("Generating topic name, please try refreshing if you keep seeing this...");
 
 	useEffect(() => {
 		axios.post('/api/dpu/pubsub', { userId }).then((response) => {
 			if (response.data.installId) {
-				setSelfHostingCode(`
-docker pull asia.gcr.io/vibi-prod/dpu/dpu &&\n
-docker run -e INSTALL_ID=${response.data.installId} asia.gcr.io/vibi-prod/dpu/dpu
-				`);
+				if (selectedInstallationType == 'individual' && selectedProvider == 'github'){
```

Comment:
Here, since you are creating the conditions anyway, you can also add that `individual + bitbucket` is "Coming soon".